base_iso3166/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# © 2016 Serv. Tecnol. Avanzados - Pedro M. Baeza
+# © 2017  Creu Blanca <www.creublanca.es>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "ISO 3166",
+    "version": "10.0.1.0.0",
+    "author": "Tecnativa, "
+              "Creu Blanca, "
+              "Eficent, "
```

Comment:
Yeah, but a simple migration like this one doesn't deserve to add two authors to the module. It would even be arguable to add one. You can however be in the contributor list in the README.